com.ibm.streamsx.topology/opt/python/packages/streamsx/rest_primitives.py

```diff
@@ -62,21 +67,25 @@ def __setattr__(self, key, value):
         super(_ResourceElement, self).__setattr__(key, value)
 
     def refresh(self):
-        """Make a REST call to refresh the JSON object stored within the object
-        Returns:
-            _ResourceElement: The provided object instance is returned with updated value
+        """Refresh the JSON representation of the resource for the latest status
```

Comment:
I thought that all those fields were pushed to the top level, so they are not expected to access `json_rep`?